UI/UI-Session9/SumUsingES6/src/app.js

```diff
@@ -0,0 +1,13 @@
+import {add} from './modules/math-functions';
+
+const result = document.getElementById('result');
+
+result.addEventListener("click", function(){
```

Comment:
@meta-shubham-jain  Add float number in his function without any error.

Reply:
Changes Updated